src/plugins/raven/selectors/getCollaborationNodes.js

```diff
@@ -0,0 +1,4 @@
+import getNode from '@triniti/cms/plugins/ncr/selectors/getNode';
+import { collaborationTopics } from '../constants';
+
+export default (state) => Object.keys(state.raven.collaboration).map((nodeRef) => (nodeRef === collaborationTopics.FAKE_ARTICLE || nodeRef === collaborationTopics.GENERAL ? null : getNode(state, nodeRef))).filter((node) => !!node);
```

Comment:
@nanetteCodes do you still need the !! filter now that we are excluding the dummy topics?